app/policies/spam_policy.rb

```diff
@@ -0,0 +1,9 @@
+class SpamPolicy < ApplicationPolicy
```

Comment:
To follow up from the other day, we've decided we'd prefer to call this `ModeratedWorkPolicy` for clarity.

Reply:
Do I need to replace all instances of “spam” with “moderatedwork” then?

Reply:
Pretty much, yup! (You should use the class `ModeratedWork` where you're currently using the symbol `:spam`, but for renaming the policy file and the `MANAGE_SPAM` constant, you should use `moderated_work_policy.rb` and `MANAGE_MODERATED_WORK` instead.)

Reply:
I guess spam_controller should also be moderated_work_controller, then?
And "Spam" in the admin banner should be "Moderated Works" instead?

Reply:
No, those can stay the same. We're just changing the policy because Pundit policies typically correspond with existing classes.